data/ecosystems/b/bnb-chain.toml

```diff
@@ -506,22 +514,61 @@ sub_ecosystems = [
 ]
 
 github_organizations = [
+  "https://github.com/AdriaGames",
+  "https://github.com/AIQ886",
+  "https://github.com/ArcX-world",
+  "https://github.com/Astra-Nova",
+  "https://github.com/Aura-Exchange",
+  "https://github.com/AutismAid",
+  "https://github.com/aveniumchain",
+  "https://github.com/BabyBakedBeans",
+  "https://github.com/BAIToken",
+  "https://github.com/Bivreost",
   "https://github.com/bnb-chain",
   "https://github.com/daohuntapp",
+  "https://github.com/chappyz",
+  "https://github.com/CookieCommunity",
+  "https://github.com/contextpilot",
+  "https://github.com/CratosTokenOfficial",
   "https://github.com/defi-org-code",
   "https://github.com/dexe-network",
+  "https://github.com/dev-four-d4",
+  "https://github.com/Diamond-Hands-App",
   "https://github.com/envoys-lab",
   "https://github.com/ERC721RA",
   "https://github.com/Everest-Option-Exchange-Team",
   "https://github.com/Fexon-Finance",
+  "https://github.com/flap-sh",
+  "https://github.com/imemessenger",
+  "https://github.com/Karacuka",
   "https://github.com/label-platform",
+  "https://github.com/Igocoin",
+  "https://github.com/invluencer",
   "https://github.com/liminal-market",
+  "https://github.com/locus-finance",
+  "https://github.com/Lorenzo-Protocol",
   "https://github.com/LuckyMachines",
-  "https://github.com/Marketplace-of-the-Multiverse",
   "https://github.com/NFTin-App",
+  "https://github.com/Marketplace-of-the-Multiverse",
+  "https://github.com/Mercury-crew",
+  "https://github.com/micro3dev",
+  "https://github.com/MintStakeShare",
+  "https://github.com/miracleplay"
+  "https://github.com/moli0618",
+  "https://github.com/oooo-protocol",
   "https://github.com/pandora-nft",
+  "https://github.com/RayFinance",
+  "https://github.com/Raw-Raw-Network",
+  "https://github.com/SleeplessAI-Lab",
+  "https://github.com/thelandlord-2023/thelandlord",
+  "https://github.com/Trendzme01",
+  "https://github.com/trump407",
+  "https://github.com/stabilityworld",
   "https://github.com/WebXCool",
-]
+  "https://github.com/WHY",
+  "https://github.com/VIALabs-io"
```

Comment:
VIALabs-io already exists an organization with this repository. It is linked to BNB Chain.

Reply:
removed VIALabs-io